### PR TITLE
Color Code Status

### LINKED ISSUE #44: Color code the task based on status
As a university student, I can colour-code tasks based on status, so that I can have an intuitive visual view of task progress.

### PR DESCRIPTION
**Pull request by noelmathewisaac**
Closes #44
Add different colors to task status to indicate progress more visually on the UI.
image

### CODE REVIEW

File: src/main/java/seedu/taskify/ui/TaskCard.java [hunk 1]
```diff
@@ -52,7 +52,19 @@ public TaskCard(Task task, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(task.getName().fullName);
         description.setText(task.getDescription().value);
-        status.setText(task.getStatus().toString());
+        String statString = task.getStatus().toString();
+        status.setText(statString);
+        System.out.println(statString);
```
Comment (khiaxeng) [inline src/main/java/seedu/taskify/ui/TaskCard.java]: Just to confirm, is System.out.println() supposed to be there?

Comment (noelmathewisaac) [reply src/main/java/seedu/taskify/ui/TaskCard.java]: Nope that was for debugging, good catch

**Review by khiaxeng — APPROVED** [review]
LGTM!

**Review by CSmortal — APPROVED** [review]
LGTM!

**Pull request merged**